client/jetpack-onboarding/main.jsx

```diff
@@ -68,16 +77,23 @@ export default connect(
 			siteId,
 			siteSlug,
 			steps,
+			userIdHashed,
 		};
 	},
 	{ recordTracksEvent },
-	( { siteId, ...stateProps }, { recordTracksEvent: recordTracksEventAction }, ownProps ) => ( {
+	(
+		{ siteId, userIdHashed, ...stateProps },
+		{ recordTracksEvent: recordTracksEventAction },
+		ownProps
+	) => ( {
 		siteId,
 		...stateProps,
 		recordJpoEvent: ( event, additionalProperties ) =>
 			recordTracksEventAction( event, {
 				blog_id: siteId,
 				site_id_type: 'jpo',
+				user_id: 'jpo_user_' + userIdHashed,
+				id: siteId + '_' + userIdHashed,
```

Comment:
Could you please elaborate on why we need both the `user_id` and `id`? Seems like we already have the `blog_id` above, and we have the `userIdHashed` contained in the `user_id`, so why do we need to to pass an additional combination of them?

Reply:
Yes, true and I mentioned this in the description of the PR. It is a recommendation by Tracks experts: it is easy to filter out events, but not to create joint events.

Reply:
Ah, cool, thanks for clarifying 👍